client/src/components/fields/select-pages.js

```diff
@@ -83,7 +89,7 @@ export class SelectPages extends React.Component {
 		const title = Title.newFromText( text, this.props.site.info );
 
 		return {
-			label: title.getNamespace().isMain() ? title.getKey().replace( /_/g, ' ' ) : `${title.getKey().replace( /_/g, ' ' )} (${title.getNamespace().getNormalizedText()})`,
+			label: title.getNamespace().isMain() ? title.getKey().replace( /_/g, ' ' ) : `${title.getNamespace().getNormalizedText()}:${title.getKey().replace( /_/g, ' ' )}`,
```

Comment:
Nit-pick: you should probably make this multi-line

Reply:
it's gone now. :)